MotDuJour/Features/Home/Sources/Home.swift

```diff
@@ -18,3 +18,20 @@ public enum HomeAction {
     case didTapFavorites
     case didTapSearch(query: String)
 }
+
+public enum HomeReducer {
```

Comment:
Nit: the other modules don't have this enum namespace

Reply:
yep, is this namespace something we want? I'm on the fence